README.md

```diff
@@ -75,8 +75,8 @@ Viewing vcperf traces in WPA requires the C++ Build Insights WPA add-in. Install
 
 | Option           | Arguments and description |
 |------------------|---------------------------|
-| `/start`         | `[/nocpusampling]` `[/level1 \| /level2 \| /level3]` `<sessionName>` |
-|                  | Tells *vcperf.exe* to start a trace under the given session name. There can only be one active session at a time on a given machine. <br/><br/> If the `/nocpusampling` option is specified, *vcperf.exe* doesn't collect CPU samples. It prevents the use of the CPU Usage (Sampled) view in Windows Performance Analyzer, but makes the collected traces smaller. <br/><br/>The `/level1`, `/level2`, or `/level3` option is used to specify which MSVC events to collect, in increasing level of information. Level 3 includes all events. Level 2 includes all events except template instantiation events. Level 1 includes all events except template instantiation, function, and file events. If unspecified, `/level2` is selected by default.<br/><br/>Once tracing is started, *vcperf.exe* returns immediately. Events are collected system-wide for all processes running on the machine. That means that you don't need to build your project from the same command prompt as the one you used to run *vcperf.exe*. For example, you can build your project from Visual Studio. |
+| `/start`         | `[/noadmin]` `[/nocpusampling]` `[/level1 \| /level2 \| /level3]` `<sessionName>` |
+|                  | Tells *vcperf.exe* to start a trace under the given session name. There can only be one active session at a time on a given machine. <br/><br/>If the `/noadmin` option is specified, *vcperf.exe* doesn't require admin privileges. This flag requires the `/nocpusampling` flag with it because CPU sampling requires admin privileges. <br/><br/> If the `/nocpusampling` option is specified, *vcperf.exe* doesn't collect CPU samples. It prevents the use of the CPU Usage (Sampled) view in Windows Performance Analyzer, but makes the collected traces smaller. <br/><br/>The `/level1`, `/level2`, or `/level3` option is used to specify which MSVC events to collect, in increasing level of information. Level 3 includes all events. Level 2 includes all events except template instantiation events. Level 1 includes all events except template instantiation, function, and file events. If unspecified, `/level2` is selected by default.<br/><br/>Once tracing is started, *vcperf.exe* returns immediately. Events are collected system-wide for all processes running on the machine. That means that you don't need to build your project from the same command prompt as the one you used to run *vcperf.exe*. For example, you can build your project from Visual Studio. |

```

Comment:
Also, the text says "_This flag requires the `/nocpusampling` flag with it because CPU sampling requires admin privileges._" The code in `Command.cpp` looks like it's ignoring `/nocpusampling` if `/noadmin` is passed, so I think it would be more accurate to say "_If the `/noadmin` option is specified, *vcperf.exe* doesn't require admin privileges, and the `/nocpusampling` flag is ignored._"

Reply:
Updated both of these comments